core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -1393,6 +1393,19 @@ object Stream {
     go(0)
   }
 
+  class PartiallyAppliedFromEither[F[_]] {
+    def apply[A](either: Either[Throwable, A])(
+        implicit ev: ApplicativeError[F, Throwable]): Stream[F, A] = {
+      val _ = ev
+      either.fold(Stream.raiseError[A], Stream.emit(_))
+    }
+  }
+
+  /**
+    * Lifts an Either[Throwable, A] to an effectful Stream
```

Comment:
Can you add example usage here following the doctest format used in other methods? Also, add punctuation to the scaladoc.

Reply:
Sure, will do.